lib/modules/manager/fleet/readme.md

```diff
@@ -0,0 +1,12 @@
+Can upgrade bundle definitions and GitRepo YAML manifests of Rancher Fleet.
+
+By default, only bundles with Helm references will be upgraded.
+To enable GitRepo updates you have to extend your [`fileMatch`](https://docs.renovatebot.com/configuration-options/#filematch) configuration.
+
+```json
+{
+  "fleet": {
+    "fileMatch": ["myGitRepoManifests\\.yaml"]
```

Comment:
Is this `\\` correct?

Reply:
yes, we need to double escape regex in strings